Automattic/wp-calypso
# Site Switcher: Update 'Add new site' button styles

Fixes #67041

Originally #67346

## Proposed Changes

Updates the styling for the 'Add new site' button in the Site Switcher.

Uses the `primary` button style.

### Calypso

<img width="483" src="https://user-images.githubusercontent.com/1500769/188759640-b992cbcb-493c-41d0-a68d-014310b0bf90.png">

### Jetpack Cloud

<img width="452" src="https://user-images.githubusercontent.com/1500769/188759394-5e9a95b4-7d94-4c14-b709-946371e8464a.png">

## Testing Instructions

1. Run Calypso locally (`yarn start`).
2. Verify the 'Add new site' button appears as expected with a variety of Dashboard Color Schemes.
3. Run Jetpack Cloud locally (`yarn start-jetpack-cloud-p`).
4. Verify the 'Add new site' button appears as expected.
